common/network-common/src/main/java/org/apache/spark/network/server/TransportChannelHandler.java

```diff
@@ -162,10 +162,11 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
         if (e.state() == IdleState.ALL_IDLE && isActuallyOverdue) {
           if (responseHandler.hasOutstandingRequests()) {
             String address = getRemoteAddress(ctx.channel());
-            logger.error("Connection to {} has been quiet for {} ms while there are outstanding " +
-              "requests. Assuming connection is dead; please adjust" +
-              " spark.{}.io.connectionTimeout if this is wrong.",
-              address, requestTimeoutNs / 1000 / 1000, transportContext.getConf().getModuleName());
+            logger.error("""
+              Connection to {} has been quiet for {} ms while there are outstanding requests. \
```

Comment:
`{}` will not be replaced by parameters in the `formatted`, right?

Reply:
Yeah. It's the format for logger. Let's revert it.